Libs/MRML/Core/vtkMRMLTransformStorageNode.cxx

```diff
@@ -258,9 +258,20 @@ int vtkMRMLTransformStorageNode::ReadFromImageFile(vtkMRMLNode *refNode)
       return 0;
     }
 
+  // ITK reads all images into LPS coordinate system (except special cases)
+  int displacementVectorCoordinateSystem = vtkMRMLStorageNode::CoordinateSystemLPS;
+
+  // NIFTI is a special case: it uses RAS coordinate system for the displacement vectors and ITK does not reorient them
+  // into LPS direction, therefore they are still in RAS.
```

Comment:
Should a link to `https://github.com/InsightSoftwareConsortium/ITK/pull/3884` be added here ? Along with a comment in `External_ITK.cmake` to remind us of updating the logic here when ITK will effectively reorient vector field values ?

Additionally, should we consider back-porting the corresponding commit to `Slicer/ITK` ?

Reply:
The ITK pull request is about accepting the "displacement vector" intent code (not just the generic "vector"). Reorientation of the vectors is just raised as a question in the pull request. I would integrate this fix now (as it is needed to be able to load VoxelMorph registration results from nifti files) and then implement the additional fixes in ITK without any time pressure.